src/lib/PlaylistCards.svelte

```diff
@@ -96,4 +98,18 @@
       opacity: 1;
     }
   }
+
+  .overridden {
+    animation: colorTransition 3s linear infinite alternate;
+  }
+
+  @keyframes colorTransition {
```

Comment:
Sorry - I think I am happy with this now though!
```suggestion
  @keyframes colorTransition {
    0%,
    100% {
      box-shadow: 0 0 7px 3px rgba(255, 215, 0, 0.4);
    }
    50% {
      box-shadow: 0 0 7px 3px rgba(255, 217, 0, 0.635);
```